app/templates/index.html

```diff
@@ -16,7 +16,6 @@
         {% with name = "college", items = colleges %}{% include "_filter.html" %}{% endwith %}
         {% with name = "year", items = years, default = "Other" %}{% include "_filter.html" %}{% endwith %}
         {% with name = "leave", items = leave, header = "Took leave?" %}{% include "_filter.html" %}{% endwith %}
-        {% with name = "eli_whitney", items = eli_whitney, header = "Eli Whitney?" %}{% include "_filter.html" %}{% endwith %}
```

Comment:
You should also remove references to `eli_whitney` under the `index` endpoint: https://github.com/Yalies/api/blob/master/app/routes.py#L45